.github/workflows/workflow.yml

```diff
@@ -77,7 +83,7 @@ jobs:
             output-name: "${{ github.workspace }}/out/wwwroot/build-info.json"
 
       - name: Upload Published Artifact
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
Can't comment on the build-info-action line (5 above), so just noting here that it should probably be v1.0.1